Trustev.WebAsync/ApiClient.cs

```diff
@@ -816,9 +864,8 @@ private static async Task<string> GetTokenAsync()
 
             TokenResponse response = await PerformHttpCallAsync<TokenResponse>(uri, HttpMethod.Post, requestJson, false, HttpRequestTimeout);
 
-            apiToken = response.APIToken;
-
-            return apiToken;
+            APIToken = response.APIToken;
```

Comment:
The call to set/get the API token is multi-threaded, if anyone is using this in a multi-threaded environment this wont work, this could be solve by simply synchronizing the access to the APIToken.
Example: https://stackoverflow.com/questions/4081986/lock-on-a-variable-in-multiple-threads